lib/settings/use-saved-settings.js

```diff
@@ -31,13 +31,13 @@ export default (settingsId, settings, setSettings, onReset) => {
 		onReset: useCallback(
 			async (e) => {
 				setSettings();
+				setSavedSettings();
 				if (e.shiftKey) {
-					await write(settingsId);
-					setSavedSettings();
+					await write(settingsId, {});
```

Comment:
This is the correct fix, but it should be done in the "driver".

```suggestion
					await write(settingsId);
```

The `local` driver handles `write` being called with a single argument, by using `localStorage.removeItem` to clean up the stored state.
https://github.com/Neovici/cosmoz-omnitable/blob/dfd7a85e1eaa66a577efd509cf3465433ee41bcf/lib/settings/drivers/local.js

You should do something similar in the `remote` driver, as that is the one with the bug (the API does not support `undefined`, but expects an empty object).
https://github.com/Neovici/cosmoz-omnitable/blob/dfd7a85e1eaa66a577efd509cf3465433ee41bcf/lib/settings/drivers/remote.js